<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kaname</class>
 <widget class="QMainWindow" name="Kaname">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kaname</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icon/cross.png</normaloff>:/icon/cross.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="LabelingGraphicsView" name="labelingView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="gbImageInfo">
          <property name="title">
           <string>Image Info</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="lblImageSize">
             <property name="text">
              <string>Image Size:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblNextObj">
             <property name="text">
              <string>Current Object</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="leImageName">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="lblImageName">
             <property name="text">
              <string>File:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="leImageSize">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="leNextObj">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI</family>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>unknown</string>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitterBottomRight">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="gbMarkedObject">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Objects</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QScrollArea" name="scrollArea">
              <property name="font">
               <font>
                <family>Microsoft YaHei UI</family>
               </font>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="objectsList">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>254</width>
                 <height>167</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="objectsVLayout"/>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cbAutoNext">
              <property name="text">
               <string>Proceed to next box &amp;automatically</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QGroupBox" name="gbKeyBindings">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>Key Bindings:</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="1">
             <widget class="QTextEdit" name="teKeyBindings">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Load Object Info: Ctrl + E&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;-&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Add Images: Ctrl + A&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Clear Images: Ctrl + W&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Save Labeling Data: Ctrl + S&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Open labeling Data: Ctrl + R&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Previous Image: V&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Next Image: B&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Delete Selection: E&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Clear Selection: Esc&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Move Current Selection: W/A/S/D&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Select Next Object: G&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Zoom In: Ctrl + Z&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Zoom Out: Ctrl + X&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Zoom Reset: Ctrl + C&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Zoom to Actual Pixels: Ctrl + V&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Grow/Shrink Horizontally: Q/E&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Grow/Shrink Vertically: Z/C&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;-&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;About: ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_EditObjectNames"/>
   <addaction name="separator"/>
   <addaction name="action_AddImages"/>
   <addaction name="action_ClearImages"/>
   <addaction name="action_Save"/>
   <addaction name="action_Open"/>
   <addaction name="separator"/>
   <addaction name="action_PreviousImage"/>
   <addaction name="action_NextImage"/>
   <addaction name="separator"/>
   <addaction name="action_DeleteSelected"/>
   <addaction name="separator"/>
   <addaction name="action_ZoomIn"/>
   <addaction name="action_ZoomOut"/>
   <addaction name="action_ZoomReset"/>
   <addaction name="action_Zoom100"/>
   <addaction name="separator"/>
   <addaction name="action_ClearSelection"/>
   <addaction name="action_SelectNextObject"/>
   <addaction name="separator"/>
   <addaction name="action_MoveLeft"/>
   <addaction name="action_MoveUp"/>
   <addaction name="action_MoveRight"/>
   <addaction name="action_MoveDown"/>
   <addaction name="separator"/>
   <addaction name="action_GrowHorizontally"/>
   <addaction name="action_GrowVertically"/>
   <addaction name="action_ShrinkHorizontally"/>
   <addaction name="action_ShrinkVertically"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_AddImages">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/add.png</normaloff>:/icon/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add Image(s)</string>
   </property>
   <property name="toolTip">
    <string>Add Image(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_ClearImages">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/clear.png</normaloff>:/icon/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear Images</string>
   </property>
   <property name="toolTip">
    <string>Clear All Images</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="action_NextImage">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/next.png</normaloff>:/icon/next.png</iconset>
   </property>
   <property name="text">
    <string>Next Image</string>
   </property>
   <property name="toolTip">
    <string>Next Image</string>
   </property>
   <property name="shortcut">
    <string>V</string>
   </property>
  </action>
  <action name="action_PreviousImage">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/previous.png</normaloff>:/icon/previous.png</iconset>
   </property>
   <property name="text">
    <string>Previous Image</string>
   </property>
   <property name="toolTip">
    <string>Previous Image</string>
   </property>
   <property name="shortcut">
    <string>B</string>
   </property>
  </action>
  <action name="action_NextObject">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/nextobj.png</normaloff>:/icon/nextobj.png</iconset>
   </property>
   <property name="text">
    <string>Mark Next Object</string>
   </property>
   <property name="toolTip">
    <string>Mark Next Object</string>
   </property>
  </action>
  <action name="action_UndoCurrentMark">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/undo.png</normaloff>:/icon/undo.png</iconset>
   </property>
   <property name="text">
    <string>Remove Last Mark</string>
   </property>
   <property name="toolTip">
    <string>Remove Last Mark</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/save.png</normaloff>:/icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save Labeling Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_EditObjectNames">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/list.png</normaloff>:/icon/list.png</iconset>
   </property>
   <property name="text">
    <string>Load Object Definitions</string>
   </property>
   <property name="toolTip">
    <string>Edit object names</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/question.png</normaloff>:/icon/question.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>?</string>
   </property>
  </action>
  <action name="action_ClearSelection">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/clearselection.png</normaloff>:/icon/clearselection.png</iconset>
   </property>
   <property name="text">
    <string>Clear Selection</string>
   </property>
   <property name="toolTip">
    <string>Clear selection</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="action_MoveLeft">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/left.png</normaloff>:/icon/left.png</iconset>
   </property>
   <property name="text">
    <string>Move Left</string>
   </property>
   <property name="toolTip">
    <string>Move Object Left</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="action_MoveRight">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/right.png</normaloff>:/icon/right.png</iconset>
   </property>
   <property name="text">
    <string>Move Right</string>
   </property>
   <property name="toolTip">
    <string>Move Object Right</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="action_MoveUp">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/up.png</normaloff>:/icon/up.png</iconset>
   </property>
   <property name="text">
    <string>Move Up</string>
   </property>
   <property name="toolTip">
    <string>Move Object Up</string>
   </property>
   <property name="shortcut">
    <string>W</string>
   </property>
  </action>
  <action name="action_MoveDown">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/down.png</normaloff>:/icon/down.png</iconset>
   </property>
   <property name="text">
    <string>Move Down</string>
   </property>
   <property name="toolTip">
    <string>Move Object Down</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="action_SelectNextObject">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/sel-next-obj.png</normaloff>:/icon/sel-next-obj.png</iconset>
   </property>
   <property name="text">
    <string>Select Next Object</string>
   </property>
   <property name="toolTip">
    <string>Select Next Object</string>
   </property>
   <property name="shortcut">
    <string>G</string>
   </property>
  </action>
  <action name="action_ZoomIn">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/zoomin.png</normaloff>:/icon/zoomin.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_ZoomOut">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/zoomout.png</normaloff>:/icon/zoomout.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_ZoomReset">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/zoomreset.png</normaloff>:/icon/zoomreset.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset zoom to fit the window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_Zoom100">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/zoom1x.png</normaloff>:/icon/zoom1x.png</iconset>
   </property>
   <property name="text">
    <string>Zoom 100%</string>
   </property>
   <property name="toolTip">
    <string>Zoom to Image Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/open.png</normaloff>:/icon/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Saved Labeling Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_DeleteSelected">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/delete.png</normaloff>:/icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete Selected</string>
   </property>
   <property name="toolTip">
    <string>Delete selected box</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="action_GrowHorizontally">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/grow_h.png</normaloff>:/icon/grow_h.png</iconset>
   </property>
   <property name="text">
    <string>Grow Horizontally</string>
   </property>
   <property name="toolTip">
    <string>Grow Horizontally</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="action_GrowVertically">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/grow_v.png</normaloff>:/icon/grow_v.png</iconset>
   </property>
   <property name="text">
    <string>Grow Vertically</string>
   </property>
   <property name="shortcut">
    <string>Z</string>
   </property>
  </action>
  <action name="action_ShrinkHorizontally">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/shrink_h.png</normaloff>:/icon/shrink_h.png</iconset>
   </property>
   <property name="text">
    <string>Shrink Horizontally</string>
   </property>
   <property name="toolTip">
    <string>Shrink Horizontally</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="action_ShrinkVertically">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/icon/shrink_v.png</normaloff>:/icon/shrink_v.png</iconset>
   </property>
   <property name="text">
    <string>Shrink Vertically</string>
   </property>
   <property name="shortcut">
    <string>C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LabelingGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>LabelingGraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
